addons/waypoint/functions/fnc_moveVehicleWp.sqf

```diff
@@ -0,0 +1,135 @@
+    private _vectorDir = _posVehicleATL vectorDiff _pos;
+    private _vectorDirNormalized = vectorNormalized _vectorDir ;
+    private _pos1 = _pos vectorAdd (_vectorDirNormalized vectorMultiply 1);
+    private _pos2 = _pos;
+    private _pos3 = _pos vectorAdd (_vectorDirNormalized vectorMultiply -1);
+    //update pos used in WUAE
+    _LeaderVehicle setvariable [QGVAR(TargetArrayMovePosition),[_pos1,_pos2,_pos3]];
+
+   //create WUAE only when needed
+if !(_LeaderVehicle getvariable [QGVAR(ActiveVehicleMove),false]) then {
+
+        _LeaderVehicle setvariable [QGVAR(ActiveVehicleMove),true];
+
+        //making group/unit easier to command
+        _driver setVariable ["lambs_danger_disableAI", true];
+        (group _driver) setVariable ["lambs_danger_disablegroupAI", true];
+
+        _driver enableAI "PATH";
+        _driver domove _pos2;
+        _driver disableAI "AUTOTARGET";
+        _driver disableAI "AUTOCOMBAT";
+        _driver disableAI "TARGET";
+        _driver disableai "COVER";
+        _driver disableai "FSM";
+        _driver setBehaviour "AWARE";
+
+        _LeaderVehicle setvariable [QGVAR(DriverWaitTime),(time + 1)];
+        _LeaderVehicle setDriveOnPath (_LeaderVehicle getvariable QGVAR(TargetArrayMovePosition));
+
+        [{
+        //condition
+            params ["_vehicle","_group"];
+            if !(waypointScript [_group,currentWaypoint _group] == QPATHTOF(functions\fnc_moveVehicleWP.sqf)) exitwith {true};
+            if !(alive (driver _vehicle)) exitwith {true};
+            if (lifestate (driver _vehicle) == "INCAPACITATED") exitwith {true};
+            if (_vehicle distance2d (_vehicle getvariable (QGVAR(TargetArrayMovePosition)) select 1) < 8 ) exitwith {true};
+            //refresh AI from "stuck" or move to updated position
+
+            if (_vehicle getvariable QGVAR(DriverWaitTime) < time) then {
+                    _vehicle setvariable [QGVAR(DriverWaitTime),(time + 2)];
+                    if (speed _vehicle < 4) then {
+                        (driver _vehicle) disableAI "Path";
+                        (driver _vehicle) enableAI "Path";
+                    };
+                    if !(isFormationLeader (driver _vehicle)) then {
+                        (driver _vehicle) doFollow leader (driver _vehicle);
+                        dostop (driver _vehicle);
+                };
+                (driver _vehicle) domove ( _vehicle getvariable QGVAR(TargetArrayMovePosition) select 1);
+                _vehicle setDriveOnPath ( _vehicle getvariable QGVAR(TargetArrayMovePosition));
+                false;
+            }
+            },{
+        //code when true
+            params ["_vehicle"];
+
+            //fixing driver behaviour where he can ignore all move orders if vehicle don't achive setDriveOnPath destinations before doFollow.
+            private _point1 = _vehicle getrelpos [2,0];
+            private _point2 = _vehicle getrelpos [3,0];
```

Comment:
Please review your whole code for such cases, there's plenty of them. Please stick to camelCase/the same casing as on Biki.
```suggestion
            private _point1 = _vehicle getRelPos [2,0];
            private _point2 = _vehicle getRelPos [3,0];
```